apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -986,7 +1500,7 @@ async function triggerEvent(
     {
       name: workflowId,
       to: {
-        subscriberId: subscriber.subscriberId,
+        subscriberId: subscriber._id,
```

Comment:
Same as above, needed to ensure the same subscriber initialized for testing is used for preference mutation and delivery.